### ISSUE TITLE
Class life and mana getting totaled

### ISSUE THREAD

**Issue by skyybender** [Eniripsa96/SkillAPI#114]
I have a class setup with one craft class and one battle class for each player. In the battle class I manage the life and mana the player should have with its current level. 
When i only join the battle class, everything works fine but when I select the craft class the life, the mana and the mana regeneration setted in the craft class is getting **added** to the players current values. 
If i set the life, mana and regeneration in the craft class to zero (0) and doing the same steps again, skill api will **reduce** my life by 10. At that point I´m out. Seems to be a little bit buggy. Thx for fixing :)

I am using the latest version of skill api premium with spigot 1.10.2 and my classes are selfcoded depending on the examples you gave in your wiki.

**Comment by Eniripsa96** [comment]
The adding is intended. The losing health part is not. I'll have to fool around with it and see if I can figure out what's going on.

**Comment by Eniripsa96** [comment]
I tried making a secondary class with 0 health and mana, but professing as it didn't change my health or mana. I may need your specific examples to figure out what's going on. Just to make sure, what SkillAPI version are you using?

**Comment by skyybender** [comment]
I just tested it with the current skill api non premium version (downloaded right now) and it is still buggy. As I told you, I have one battle and one craft class category. The craft classes are set up with 0 health and 0 mana. When I profess the craft class **only** and my health get reduced by 10. Now I reset the classes again and first profess the battle class, that makes me having 26 health. When I profess my second craft class with still having the battle class I get 10 more health, even if I set the health and mana to 0. Sry I don´t get it, hope you understand my problem now.
